File: SQLite_Example/app/src/main/java/com/example/sqlite/juan/activities/MainActivity.java
package com.example.sqlite.juan.activities;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

import com.example.sqlite.juan.R;

public class MainActivity extends AppCompatActivity {
    Button insert;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        insert =(Button) findViewById(R.id.insert);

        insert.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, InsertActivity.class);
                startActivity(intent);
            }
        });
    }

}

File: SQLite_Example/app/src/main/java/com/example/sqlite/juan/activities/MainActivity.kt
package com.example.sqlite.juan.activities

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.example.sqlite.juan.R

class MainActivity : AppCompatActivity() {
    var insert: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        insert = findViewById<View>(R.id.insert) as Button
        insert!!.setOnClickListener {
            val intent = Intent(this@MainActivity, InsertActivity::class.java)
            startActivity(intent)
        }
    }
}